pages/_en-US/gbarunner2/bios-dump.md

```diff
@@ -0,0 +1,81 @@
+---
+lang: en-US
+layout: wiki
+section: gbarunner2
+title: GBA BIOS Dumping
+description: How to extract the Nintendo GBA BIOS from your console
+tabs:
+  - tab-3ds-sd-card: 3DS with open_agb_firm
+    tab-gba-flashcart: GBA/DS/DS Lite with a GBA-mode flashcart
```

Comment:
```suggestion
  - 3ds-sd-card: 3DS with open_agb_firm
    gba-flashcart: GBA/DS/DS Lite with a GBA-mode flashcart
```
These should not have `tab-`, that's added automatically

Reply:
So... the forwarder guide had this ever since the beginning.

Then I forked the forwarder guide into the DS BIOS/FW guide.

Then I forked _that_ to this guide.

We should do this in a separate commit, fix all other pages while at it.